apps/foundation-demo/src/app/cart/components/cart-summary/cart-summary.component.spec.ts

```diff
@@ -58,52 +58,52 @@ describe('CheckoutCartComponent', () => {
     });
 
     wrapper.cartValue = mockCart;
-    wrapper.subtitleValue = stubSubtitle;
+    wrapper.titleValue = stubTitle;
 
     fixture.detectChanges();
 
-    checkoutCartItems = fixture.debugElement.queryAll(By.css('demo-checkout-cart-item'));
-    checkoutCart = fixture.debugElement.query(By.css('demo-checkout-cart')).componentInstance;
+    cartSummaryItems = fixture.debugElement.queryAll(By.css('demo-minicart-item'));
+    cartSummary = fixture.debugElement.query(By.css('demo-cart-summary')).componentInstance;
   });
 
   it('should create', () => {
-    expect(checkoutCart).toBeTruthy();
+    expect(cartSummary).toBeTruthy();
   });
 
   it('can be passed a Cart object', () => {
-    expect(checkoutCart.cart).toEqual(mockCart);
+    expect(cartSummary.cart).toEqual(mockCart);
   });
 
-  it('can be passed a subtitle as input', () => {
-    expect(checkoutCart.subtitle).toEqual(stubSubtitle);
+  it('can be passed a title as input', () => {
+    expect(cartSummary.title).toEqual(stubTitle);
   });
 
-  it('renders a <demo-checkout-cart-item> for every checkoutCart.items', () => {
-    expect(checkoutCartItems.length).toEqual(mockCart.items.length);
+  it('renders a <demo-minicart-item> for every miniCart.items', () => {
+    expect(cartSummaryItems.length).toEqual(mockCart.items.length);
   });
 
-  describe('on <demo-checkout-cart-item>', () => {
+  describe('on <demo-minicart-item>', () => {
     it('should set item', () => {
-      expect(checkoutCartItems[0].componentInstance.item).toEqual(mockCart.items[0]);  
+      expect(cartSummaryItems[0].componentInstance.item).toEqual(mockCart.items[0]);  
     });
   });
 
-  describe('when subtitle is null', () => {
+  describe('when title is null', () => {
     
-    it('should not render .checkout-cart__title', () => {
-      checkoutCart.subtitle = null;
+    it('should not render .cart-summary__title', () => {
+      cartSummary.title = null;
       fixture.detectChanges();
 
-      const cartTitleElement = fixture.debugElement.query(By.css('.checkout-cart__title'));
+      const cartTitleElement = fixture.debugElement.query(By.css('.cart-summary__title'));
 
       expect(cartTitleElement).toBeNull();
     });
   });
 
-  describe('when subtitle is defined', () => {
+  describe('when title is defined', () => {
     
-    it('should render .checkout-cart__title', () => {
-      const cartTitleElement = fixture.debugElement.query(By.css('.checkout-cart__title'));
+    it('should render .cart-summary__title', () => {
```

Comment:
should be "should render .cart-summary__header"

Reply:
Resolved in 08c6569